src/tools/miri/tests/native-lib/ptr_read_access.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+
+/* Test. */
```

Comment:
Please also state the "name" of the test in these comments (matching the name of their function in the Rust file).

Reply:
Of course - unfortunately slipped by correction morning - I added the Rust test names for each section.